### PR TITLE
Openid prompt

### PR DESCRIPTION
**Pull request by fvztdk**
Add an enviroment variable to configure OpenId prompt parameter; keep the default value as 'login'

Pairing with @fejj182

### CODE REVIEW

**Comment by moufmouf** [comment]
Hi @fvztdk and @fejj182!

Thanks a lot for this new feature!

- [ ] Could I ask you to fix the conflict that appeared (I worked quite a lot on the OpenID feature these days)
- [ ] Also, could you document the new environment variables in the new "/docs/dev/openid.md" file?
- [ ] Finally, could you also modify the /contrib/docker/docker-compose.prod.yaml file?

Thanks again!

File: .env.template [hunk 1]
```diff
@@ -34,6 +34,7 @@ OPID_CLIENT_ISSUER=
 OPID_CLIENT_REDIRECT_URL=
 OPID_LOGIN_SCREEN_PROVIDER=http://pusher.workadventure.localhost/login-screen
 OPID_PROFILE_SCREEN_PROVIDER=
+OPID_PROMPT=
```
Comment (gparant) [inline .env.template:37]: Maybe add `login` in the `.env.template` file?

**Comment by fvztdk** [comment]
Hi, been out for the last few days I have done the recommend changes

- [x]  Could I ask you to fix the conflict that appeared (I worked quite a lot on the OpenID feature these days)
- [x] Also, could you document the new environment variables in the new "/docs/dev/openid.md" file?
- [x] Finally, could you also modify the /contrib/docker/docker-compose.prod.yaml file?
- [x]  .env.template

**Comment by moufmouf** [comment]
Nice, thanks a lot! :heart:

**Pull request merged**